algebird-core/src/main/scala/com/twitter/algebird/MomentsGroup.scala

```diff
@@ -29,14 +29,23 @@ case class Moments(m0: Long, m1: Double, m2: Double, m3: Double, m4: Double) {
   def mean = m1
 
   // Population variance, not sample variance.
-  def variance = m2 / count
+  def variance = if (count > 1)
```

Comment:
If folks have some need for the statistics in the case of a very low count, then adding `varianceUnsafe` (etc.) makes sense. (But when would these numbers be of value? By definition, the statistics aren't defined in these edge cases.)

On the other hand, if the purpose for `varianceUnsafe` is for backward compatibility, then maybe it would be better to instead dump my changes into a `varianceSafe` (and similar), and keep the current definitions as they already were. That is to say, the existing definitions are already the unsafe versions (since new algorithms might change their values when the count is very low), and the pull request is creating the safe versions.

Other reasons for making that change that I missed?